org.chromium.Chromium.yaml

```diff
@@ -64,6 +64,8 @@ sdk-extensions:
   - org.freedesktop.Sdk.Extension.openjdk11
 
 modules:
+  - pipewire.json
```

Comment:
Can the body of this just be inline (& right below `libsecret.json`)?

Reply:
inlined this.